docs/user/dashboard/create-dashboards.asciidoc

```diff
@@ -155,6 +156,24 @@ Copy panels from one dashboard to another dashboard.
 [role="screenshot"]
 image:https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt64206db263cf5514/66f49286833cffb09bebd18d/copy-to-dashboard-8.15.0.gif[Copy a panel to another dashboard, width 30%]
 
+[[duplicate-dashboards]]
+== Duplicate dashboards
+
+. Open the dashboard you want to duplicate.
+
+. In *View* mode, click *Duplicate* in the toolbar.
+
+. In the *Duplicate Dashboard* window, enter a title and optional description and tags.
```

Comment:
```suggestion
. In the *Duplicate dashboard* window, enter a title and optional description and tags.
```
Looks like it's sentence case in the product

Reply:
Good catch!